TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -329,6 +335,70 @@ public static void ScanTokensLineContinuationGroup(IList<TokensLine> continuatio
+                            char endDelimiter = APOSTROPHE;
+                            bool isBadEndDelimiter = false;
+                            if (isLastLine)
+                            {
+                                // check closing delimiter of the last continuation line
+                                string lastLine = continuationLine.SourceText.TrimEnd();
+                                int pos = lastLine.LastIndexOf(startDelimiter);
+                                if (pos > lastLine.IndexOf(startDelimiter))
+                                {
+                                    // delimiter is also present near the end
+                                    lastLine = lastLine.Substring(0, pos + 1);
+                                }
+                                else
+                                {
+                                    // . is present at the end
+                                    pos = lastLine.Length - 1;
+                                    if (lastLine[pos] == '.')
+                                    {
+                                        lastLine = lastLine.Substring(0, pos).TrimEnd();
+                                    }
+                                }
+
+                                endDelimiter = lastLine[lastLine.Length - 1];
+                                isBadEndDelimiter = endDelimiter != lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter;
+                            }
+
+                            if (isBadStartDelimiter || isBadEndDelimiter)
+                            {
+                                if (startDelimiter != APOSTROPHE && startDelimiter != QUOTATION)
+                                {
+                                    // no valid starting delimiter
+                                    continuationLine.AddDiagnostic(MessageCode.SyntaxErrorInParser,
+                                        startOfContinuationIndex, startOfContinuationIndex + 1, 
+                                        "Starting delimiter of the continuation line is missing.");
+                                    offsetForLiteralContinuation = 0;
+                                }
+                                else if (endDelimiter != APOSTROPHE && endDelimiter != QUOTATION)
+                                {
+                                    // no valid closing delimiter
+                                    continuationLine.AddDiagnostic(MessageCode.SyntaxErrorInParser,
+                                        startOfContinuationIndex, startOfContinuationIndex + 1,
+                                        "Closing delimiter of the continuation line is missing.");
+                                    offsetForLiteralContinuation = 0;
+
+                                }
+                                else
+                                {
+                                    // different delimiters
```

Comment:
Not much but I would add "between start and end delimiters" in comment just to be clearer

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1604/commits/86f9e27e5ebf7313470d0b3ae58e3e9ea7a080d6